snippets/storage-next/emulator-suite/storage_emulator_connect.js

```diff
@@ -0,0 +1,15 @@
+// This snippet file was generated by processing the source file:
+// ./storage-next/emulator-suite.js
+//
+// To update the snippets in this file, edit the source and then run
+// 'npm run snippets'.
+
+// [START storage_emulator_connect_modular]
+import { getStorage, connectStorageEmulator } from "firebase/storage";
+
+const storage = getStorage();
+if (location.hostname === "localhost") {
+  // Point to the RTDB emulator running on localhost.
```

Comment:
Did you mean storage emulator?
```suggestion
  // Point to the Storage emulator running on localhost.
```